app/controllers/smart_proxies_controller.rb

```diff
@@ -8,6 +8,7 @@ def index
   end
 
   def show
+    render_extensible
```

Comment:
I don't understand this rendering approach. Why you override the defualt render method? Isn't better just to add before_filter which simply finds the `@page` class variable so it is available in the template? That looks cleaner to me, there might be hidden dragon you want to explain here.